### PR TITLE
Add newsletter signup example

### PR DESCRIPTION
**Pull request by hidalgopl**
Hey @zodman @jonathan-s  :hand: 
I just tried sockpuppet and wanted to understand how I can effectively use forms to persist a state in DB, so I added this simple newsletter signup example which covers Create/Read/Update/Delete operation using reflexes.

Please let me know if anything here could be done better / simpler, as I'm new sockpuppet user :) Thanks!

### CODE REVIEW

**Comment by zodman** [comment]
@hidalgopl  simple example thanks!

For accept your PR  plz use the existing template structure and add the link to the menu. without it no body will see you your example.

**Comment by hidalgopl** [comment]
thanks @zodman ! Edited templates as you suggested :+1:

File: core/views/mixins.py [hunk 1]
```diff
@@ -59,12 +60,21 @@ class ChatMixin(MixinBase):
         ('core/templates/_chat_demo.html', 'html', 'htmldjango'),
     )
 
+
 class CalendarMixin(MixinBase):
     files = (
         ('core/views/calendar.py', 'python', 'python3'),
-#        ('core/reflexes/calendar_reflex.py', 'python', 'python3'),
-#        ('core/javascript/controllers/calendar_controller.js', 'javascript', 'javascript'),
+       # ('core/reflexes/calendar_reflex.py', 'python', 'python3'),
```
Comment (zodman) [inline core/views/mixins.py]: remove not used code plz

Comment (hidalgopl) [reply core/views/mixins.py]: :+1: removed

**Pull request merged**